Pokedex/MainScreen.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ViewController.swift
+//  Pokedex
+//
+//  Created by Стефан  Миливоевич on 5.10.22..
+//
+
+import UIKit
+
+class MainScreen: UIViewController, UITableViewDelegate, UITableViewDataSource  {
+   
+    
+    
+    var data = [String()]
```

Comment:
Возможно так более наглядно:
```suggestion
    var data: [String] = []
```